types/knex/index.d.ts

```diff
@@ -139,12 +139,27 @@ declare namespace Knex {
 
         // Aggregation
         count(columnName?: string): QueryBuilder;
+        count(columnName: {[key: string]: string}): QueryBuilder;
```

Comment:
Seems like you should aim to consolidate these into one signature:

```ts
coount(columnName?: string | Record<string, string> | Record<string, string[]> | Record<string, Knex.Raw>): QueryBuilder;
```

Better yet, is this more accurate?

```ts
coount(columnName?: string | Record<string, string | string[] | Knex.Raw>): QueryBuilder;
```